set-link/entrypoint.sh

```diff
@@ -55,20 +62,19 @@ else
 fi
 
 if [[ -n "$story" ]]; then
-    new_body=${body/$STORY_LINK_TEXT/$link_url} # replace the story link text
+  new_body=${body/$STORY_LINK_TEXT/$link_url} # replace the story link text
 fi
 
 # If we still don't have the url in the body, tack it on the beginning
-if [[ "$new_body" != *"$link_url"* ]] && [[ -n "$story" ]] ; then
-    new_body="$link_url\n$new_body"
+if [[ "$new_body" != *"$link_url"* && -n "$story" ]]; then
+  new_body="$link_url\n$new_body"
 fi
 
 new_title="${title}"
 
-formatted_title=`echo "${new_title}" | cut -d "/" -f 3`
+formatted_title="$(cut <<<"${new_title}" -d "/" -f3)"
```

Comment:
@cdelst what's the reason for this bit? This is causing a title like: `sc-187720 - update aws version/service_legacy tag for applications/data-exporter` to become `[sc-187720] Data-exporter`

Reply:
Never mind, I'm guessing this is for the ones where it picks up the branch name as a title. I've addressed it.